modules/vaos/app/controllers/vaos/v2/appointments_controller.rb

```diff
@@ -96,6 +71,10 @@ def update
 
       private
 
+      def set_facility_error_msg(appointment)
```

Comment:
Keeping this logic in controllers since moving it to services broke mobile. I think they have the expectation that location is set to nil on failure instead of the error message.

Reply:
I checked with the mobile team and that expectation is due to their business needs so we can't break this behavior for the time being.